src/app/browse-by/browse-by-geospatial-data/browse-by-geospatial-data.component.ts

```diff
@@ -0,0 +1,110 @@
+  getFirstSucceededRemoteDataPayload,
+} from '../../core/shared/operators';
+import { SearchService } from '../../core/shared/search/search.service';
+import { SearchConfigurationService } from '../../core/shared/search/search-configuration.service';
+import { hasValue } from '../../shared/empty.util';
+import { GeospatialMapComponent } from '../../shared/geospatial-map/geospatial-map.component';
+import { FacetValues } from '../../shared/search/models/facet-values.model';
+import { PaginatedSearchOptions } from '../../shared/search/models/paginated-search-options.model';
+
+@Component({
+  selector: 'ds-browse-by-geospatial-data',
+  templateUrl: './browse-by-geospatial-data.component.html',
+  styleUrls: ['./browse-by-geospatial-data.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [GeospatialMapComponent, NgIf, AsyncPipe, TranslateModule],
+  standalone: true,
+})
+/**
+ * Component displaying a large 'browse map', which is really a geolocation few of the 'point' facet defined
+ * in the geospatial discovery configuration.
+ * The markers are clustered by location, and each individual marker will link to a search page for that point value
+ * as a filter.
+ *
+ * @author Kim Shepherd
+ */
+export class BrowseByGeospatialDataComponent implements OnInit {
+
+  protected readonly isPlatformBrowser = isPlatformBrowser;
+
+  public facetValues$: Observable<FacetValues> = of(null);
+
+  constructor(
+    @Inject(PLATFORM_ID) public platformId: string,
+    private searchConfigurationService: SearchConfigurationService,
+    private searchService: SearchService,
+    protected route: ActivatedRoute,
+  ) {}
+
+  public scope$: Observable<string> ;
+
+  ngOnInit(): void {
+    this.scope$ = this.route.queryParams.pipe(
+      map((params: Params) => params.scope),
+    );
+    this.facetValues$ = this.getFacetValues();
+  }
+
+  /**
+   * Get facet values for use in rendering 'browse by' geospatial map
+   */
+  getFacetValues(): Observable<FacetValues> {
+    return combineLatest([this.scope$, this.searchConfigurationService.getConfig(
+      // If the geospatial configuration is not found, default will be returned and used
+      '', environment.geospatialMapViewer.spatialFacetDiscoveryConfiguration).pipe(
+      getFirstCompletedRemoteData(),
+      getFirstSucceededRemoteDataPayload(),
+      filter((searchFilterConfigs) => hasValue(searchFilterConfigs)),
+      take(1),
+      map((searchFilterConfigs) => searchFilterConfigs[0]),
+      filter((searchFilterConfig) => hasValue(searchFilterConfig))),
+    ],
+    ).pipe(
+      switchMap(([scope, searchFilterConfig]) => {
+        // Get all points in one page, if possible
+        searchFilterConfig.pageSize = 99999;
+        const searchOptions: PaginatedSearchOptions = Object.assign({
+          'configuration': environment.geospatialMapViewer.spatialFacetDiscoveryConfiguration,
+          'scope': scope,
+          'facetLimit': 99999,
```

Comment:
Same here, isn't this a possible performance issue? We just recently tried to remove these sort of `99999` references from components in #2513

Reply:
see previous comment - a warning is a good idea, but in order to make a browse map usable without a lot of complex solr spatial features, we do need to be able to make large queries.
This is a facet query, the values are lists of facet string values and counts, not full indexable search objects as with a regular discovery search.
So I hope this is good enough for an initial implementation